docs/source/building_an_experiment.rst

```diff
@@ -4,14 +4,227 @@ Building an Experiment
+integration time rather than run concurrently.
+
+**3. INTEGRATION**   
+
+Integration interfacing allows for pulse sequences defined in the slices to alternate sequence by sequence each other within an integration period. Slices which are interfaced in this manner must share the same INTT and INTN values for this to work. It's important to remember that each sequence 
+only averages with sequences from the same slice. 
+
+**4. PULSE**   
+
+Pulse interfacing allows for pulse sequences to be run together concurrently. Slices will have their pulse sequences mixed and layered together so that the data transmits at the same time. Slices of different frequencies can be 
+mixed simultaneously and slices of different pulse sequences can also run together at the cost of having more blanked samples.
+
+Let's look at some examples of common experiments that can easily be separated into multiple slices. 
+
+In a CUTLASS-style experiment, the pulse in the sequence is actually two pulses of differing transmit frequency. This is a 'quasi'-simultaneous multi-frequency experiment. To build this experiment, two slices can be PULSE interfaced. The pulses from both slices are combined into a single sequence and data from those integrations are used for both slices (filtering the raw data separates the frequencies). 
+
+.. image:: img/cutlass.png
+   :width: 800px
+   :alt: CUTLASS-style experiment slice interfacing 
+   :align: center
+
+In a themisscan experiment, a single beam is interleaved with a full scan. The beam_order can be unique to different slices, and these slices could be INTTIME interfaced to separate the camping beam data from the full scan,
+if desired. With INTTIME interfacing, one averaging period of one slice will be followed by an averaging period of another, and so on. The averaging periods are interleaved. The resulting experiment runs beams 0, 7, 1, 7, etc. 
+
+.. image:: img/themisscan.png
+   :width: 800px
+   :alt: THEMISSCAN slice interfacing 
+   :align: center
+
+In a twofsound experiment, a full scan of one frequency is followed by a full scan of another frequency. The txfreq are unique between the slices. In this experiment, the slices are SCAN interfaced. A full scan of slice 0 runs 
+followed by a full scan of slice 1, and then the process repeats. 
+
+.. image:: img/twofsound.png
+   :width: 800px
+   :alt: TWOFSOUND slice interfacing 
+   :align: center
+
+
+Here's a theoretical example showing all types of interfacing. In this example, slices 0 and 1 are PULSE interfaced. Slices 0 and 2 are INTEGRATION interfaced. Slices 0 and 3 are INTTIME interfaced. Slices 0 and 4 are SCAN interfaced.
+
+.. image:: img/one-experiment-all-interfacing-types.png
+   :width: 800px
+   :alt: An example showing all types of slice interfacing 
+   :align: center
+
+Slice Keys
+----------
+A slice is defined by the user as a dictionary, with the following preset keys:
+
```

Comment:
Data types could be added to all of these preset key descriptions (list of integers, integer, float, etc)